### ISSUE TITLE
World border is diameter, not radius

### ISSUE THREAD

**Issue by TitaneerYeager** [LemADEC/WarpDrive#298]
Hi,
Thank you for tanking time to read this. With your mod installed on minecraft 1.7.10 there is a world border in the overworld. Is there a way to disable this? If not can you add a way too? One of the top reasons I refuse to update any higher than 1.7.10 is because a world border was added in vanilla versions 1.8 and up. Plus there are a lot of "dead" mods that will not be updating any higher versions that I like. I am also mentioning 1.7.10, becuase that is the version of warpdrive I am using. (warpdrive-1.7.10-1.3.34-.jar) Thank you.

**Comment by LemADEC** [comment]
There's always a world border by design of the vanilla game itself. You can already adjust planet position/sizes to have larger border in any planet per said.
You can not and never will have 'infinite' world border.

**Comment by TitaneerYeager** [comment]
When I removed your mod the world border was gone. So either there is no world border in 1.7.10 (which I researched(on the official minecraft wiki) and found the world border was added in 1.8), or it is(in vanilla) SUPERBLY LARGER than what your mod makes. And if the latter is true, why can't I find anything about 1.7.10 and the world border? Also you said about making the world border larger per planet, so do you mean in the warp drive config file? If so I do not understand it because I set the RADIUS of the overworld to 10000, and the world border was at 2000, 2000 to -2000, -2000! That is a 4000 DIAMETER! NOT RADIUS, and I set the radius to 10000! not 4000! So according to the config, shouldn’t the world border be at 10000 10000 to -10000, -10000?!
thank you.

**Comment by LemADEC** [comment]
Vanilla world border is at ±30000000.
The whole universe, including all planets needs to fit in the range.
The border size isn't the radius. Neither the comments nor the labels are saying it is radius.
You didn't share your modified configuration but you probably changed the wrong parameter or another dimension.

**Comment by TitaneerYeager** [comment]
Ok, under Planets section of the configuration file I have:(WarpDrive.cfg) (I had to put the quotes around the hashtag or the after text would be enlarged in the github comment)
"#" dimensionId, dimensionCenterX, dimensionCenterZ, radiusX, radiusZ, spaceCenterX, spaceCenterZ
    I:overworld <
        0
        0
        0
        10000
        10000
        0
        0
     >
And yes it does say RADIUS see radiusX, raduisY, above, which is the only thing I changed in the config.(the size of radiusX and radiusZ to 10000 instead of I think it was 1000)
About the vanilla world border, thank you for telling me that! But that brings up a new question: Why does the "Official" minecraft wiki say that it was added in 1.8? So is it possible to enlarge your world border? If so where? thank you!
